python/paddle/fluid/layers/detection.py

```diff
@@ -586,12 +586,13 @@ def multi_box_head(inputs,
        name(str): Name of the prior box layer. Default: None.
 
     Returns:
-        mbox_loc(list): The predicted boxes' location of the inputs.
-             The layout of each element is [N, H, W, Priors]. Priors
+        mbox_loc(Variable): The predicted boxes' location of the inputs.
+             The layout of each element is [N, H*W*Priors, 4]. Priors
```

Comment:
The layout of each element is [N, HWPriors, C]. Priors

->

The layout is [N, HWPriors, C], where Priors xxx

Reply:
done